spec/models/topic_spec.rb

```diff
@@ -580,6 +580,11 @@ def build_topic_with_title(title)
       end
     end
 
+    it 'does not result in a syntax error when removing accents' do
+      SiteSetting.search_ignore_accents = true
+      expect(Topic.similar_to('something', 'it\'s')).to eq([])
```

Comment:
```suggestion
      expect(Topic.similar_to('something', "it's")).to eq([])
```

If you use double quotes you don't need to escape the single quote in the string.